docs/release_checklist.md

```diff
@@ -0,0 +1,85 @@
+# Release checklist
+
+## New feature release
+
+* Update `WhatsNew.txt`
+
+* Bump version number to 2.EVEN.0 in all these locations:
+
+    * `configure.ac`, `CMakeLists.txt`: `SDL_*_VERSION`
+    * `Xcode/SDL/Info-Framework.plist`: `CFBundleShortVersionString`,
+        `CFBundleVersion`
+    * `Makefile.os2`: `VERSION`
```

Comment:
`DYLIB_CURRENT_VERSION` (and `DYLIB_COMPATIBILITY_VERSION`) in Xcode and CMakeLists.txt

Reply:
They're in the next section, as part of the ABI versioning rather than the "marketing" versioning.

Reply:
Sorry, my bad.